src/Header/Nav/NavLinks/NavLinks.jsx

```diff
@@ -136,14 +137,41 @@ export function NavLinks({
 										)}
 									</button>
 								) : (
-									<a
-										href={href}
-										aria-current={ariaCurrent}
-										className={styles.link}
-										onClick={handleNavLinkClick}
-									>
-										<span aria-label={abbreviation && title}>{text}</span>
-									</a>
+									<>
+										<a
+											href={href}
+											aria-current={ariaCurrent}
+											className={styles.link}
+											onClick={handleNavLinkClick}
+										>
+											<span aria-label={abbreviation && title}>{text}</span>
+											{nestedLinks && (
+												<ChevronDown
+													className={styles.icon}
+													pointerEvents="none"
+												/>
+											)}
+										</a>
+
+										{nestedLinks && (
+											<>
+												<ul
+													id={id}
+													className={styles.nonJsDropdown}
+													aria-label="More NICE services"
+												>
+													{nestedLinks &&
```

Comment:
do you need to null check nestedLinks again here as it's got an outer check already?